src/tools/kata-ctl/Cargo.toml

```diff
@@ -10,5 +10,6 @@ authors = ["The Kata Containers community <kata-dev@lists.katacontainers.io>"]
 edition = "2021"
 
 [dependencies]
-clap = "2.33.0"
 anyhow = "1.0.31"
+clap = "2.33.0"
+semver = "1.0.12"
```

Comment:
Can you move this to a new `[dev-dependencies]` block? That way, the `semver` crate will only be imported for the unit tests (see https://github.com/kata-containers/kata-containers/blob/main/docs/presentations/unit-testing/kata-containers-unit-testing.md#writing-new-tests-specific-advice-part-3)

Reply:
done, thank you